src/server/pegasus_server_impl.h

```diff
@@ -413,11 +425,18 @@ class pegasus_server_impl : public pegasus_read_service
     std::shared_ptr<KeyWithTTLCompactionFilterFactory> _key_ttl_compaction_filter_factory;
     std::shared_ptr<rocksdb::Statistics> _statistics;
     rocksdb::DBOptions _db_opts;
+    // The value of option in data_cf according to conf template file config.ini
     rocksdb::ColumnFamilyOptions _data_cf_opts;
+    // Dynamically calculate the value of current data_cf option according to the conf module file
+    // and usage mode
+    rocksdb::ColumnFamilyOptions _table_data_cf_opts;
     rocksdb::ColumnFamilyOptions _meta_cf_opts;
     rocksdb::ReadOptions _data_cf_rd_opts;
     std::string _usage_scenario;
     std::string _user_specified_compaction;
+    // Whether it is necessary to update the current data_cf, it is required when opening the db,
+    // but not later
+    bool _table_data_cf_opts_recalculated;
```

Comment:
the name `_table_data_cf_opts_recalculated` means option has been **recalculated**, it's different with the meaning you defined before, that is to say, true and flase should reverse.